packages/ui/src/components/forms/file-upload/FileUpload.vue

```diff
@@ -118,6 +120,10 @@ export const ForgeFileUpload = /*#__PURE__*/ (
     maxFileInput: {
       type: Number,
       default: null
+    },
+    editableFileName: {
+      type: Boolean,
+      default: true
```

Comment:
wonder if we should keep the default false to maintain "backwards" compatibility, in case people don't actually want their filenames editable, if that makes sense

Reply:
Good call dont wanna make breaking changes

Reply:
Yeah I think I forgot to change this back to default